config/rubocop/1.9.1/rubocop-2.7.yml

```diff
@@ -126,6 +132,13 @@ Style/ClassVars:
 
 Style/CollectionMethods:
   Enabled: true
+  PreferredMethods:
+    collect: 'map'
+    collect!: 'map!'
+    inject: 'reduce'
+    find: 'detect'
```

Comment:
Only change here since there seems to be a lot more `detect` over `find` throughout the existing code